runners/flink/1.15/src/main/java/org/apache/beam/runners/flink/transform/sql/SqlTransform.java

```diff
@@ -114,6 +114,16 @@ public static <T> SingleOutputSqlTransform<T> of(Class<T> outputClass) {
     return new SingleOutputSqlTransform<>(of(Integer.class, outputClass));
   }
 
+  /**
+   * Create a {@link StatementOnlySqlTransform} which takes a full script of SQL statements and
+   * execute them. The statements must have at least one <code>INSERT INTO</code> statement.
```

Comment:
@becketqin Sorry for commenting after the PR is merged.


1. What if the SQL statements had a `INSERT OVERWRITE` instead of `INSERT INTO`? Is that not a valid `StatementOnlySqlTransform`?
2. Similarly, how about having a `CREATE TABLE AS SELECT` with out `INSERT INTO`, should we support that as well?

Reply:
That is a good point. We should support both. I'll have a follow up patch.

Reply:
I created the [PR](https://github.com/linkedin/beam/pull/116). However, currenlty we cannot support CTAS because the OSS Flink `StreamStatementSet.attachAsDataStream()` does not support that yet. We will need to make change to the OSS Flink first.